src/session_impl.cpp

```diff
@@ -5522,11 +5525,11 @@ namespace {
 		if (m_listen_sockets.empty()) return 0;
 		if (sock)
 		{
-			// if we're using a proxy, we won't be able to accept any TCP
-			// connections. Not even uTP connections via the port we know about.
```

Comment:
I think it would be better to update or extend this comment, rather than removing it